### PR TITLE
ops-bedrock: change devnet l1 init holder

### PR DESCRIPTION
**Pull request by welkin22**
### Description
Modify the init_holder of L1 in devnet to be the same as L2

### Rationale
Modify init_holder uniformly to 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266, so that users only need one account during testing.

### Example
After changing:  

L1 test account:

address: 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
Private key: ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80

L2 test account:

Address: 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
Private key: ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80

### Changes

Notable changes:
* Modify the init_holder of L1 in devnet to be the same as L2
* The code that modifies deploy_config should be executed earlier

### CODE REVIEW

File: ops-bedrock/l1-entrypoint.sh [hunk 1]
```diff
@@ -15,6 +15,9 @@ cd genesis
 npm install
 cd ..
 
+sed -i -e "s/INIT_HOLDER=\"0x04d63aBCd2b9b1baa327f2Dda0f873F197ccd186\"/INIT_HOLDER=\"$INIT_HOLDER\"/g" .env
+sed -i -e "s/INIT_HOLDER_PRV=\"59ba8068eb256d520179e903f43dacf6d8d57d72bd306e1bd603fdb8c8da10e8\"/INIT_HOLDER_PRV=\"$INIT_HOLDER_PRV\"/g" .env
```
Comment (owen-reorg) [inline ops-bedrock/l1-entrypoint.sh:19]: Why is this necessary?  It seems `ops-bedrock/l1.env` already changed this

Comment (welkin22) [reply ops-bedrock/l1-entrypoint.sh:19]: l1.env just injects the value into the environment variable, but in the node-deploy repo of bsc, it will use its own .env to overwrite the environment variable, causing the modification of l1.env to become invalid.

File: bedrock-devnet/devnet/__init__.py [hunk 2]
```diff
@@ -127,6 +127,25 @@ def devnet_deploy(paths):
     devnet_cfg_orig = pjoin(paths.contracts_bedrock_dir, 'deploy-config', 'devnetL1.json')
     devnet_cfg_backup = pjoin(paths.devnet_dir, 'devnetL1.json.bak')
     shutil.copy(devnet_cfg_orig, devnet_cfg_backup)
+    deploy_config = read_json(devnet_cfg_orig)
+    l1BlockTag = l1BlockTagGet()["result"]
+    log.info(l1BlockTag)
```
Comment (bendanzhentan) [inline bedrock-devnet/devnet/__init__.py:132]: Should we remove this log since it seems useless for users?

File: bedrock-devnet/devnet/__init__.py [hunk 3]
```diff
@@ -127,6 +127,25 @@ def devnet_deploy(paths):
...
+    l1BlockTimestamp = l1BlockTimestampGet(l1BlockTag)["result"]["timestamp"]
+    log.info(l1BlockTimestamp)
```
Comment (bendanzhentan) [inline bedrock-devnet/devnet/__init__.py:134]: ditto

**Review by bendanzhentan — APPROVED** [review]

File: bedrock-devnet/devnet/__init__.py (hunk 2 above)
Comment (welkin22) [reply bedrock-devnet/devnet/__init__.py:132]: No, we need this log to know the situation of L1
